content/docs/gke/authentication.md

```diff
@@ -0,0 +1,175 @@
++++
+title = "Authenticating Kubeflow to GCP"
+description = "Troubleshooting guide for authentication and authorization to GCP"
+weight = 4
++++
+
+
+# Authenticating gcloud
+
+[The `gcloud` tool](https://cloud.google.com/sdk/gcloud/) is used to interact with Google Cloud Platform (GCP) over the command line. 
+It can be used to [set up Google Kubernetes Engine (GKE) clusters](https://cloud.google.com/sdk/gcloud/reference/container/clusters/create), 
+and interact with other Google services.
+
+### Logging in
+
+There are two ways to authenticate the gcloud command:
+
+
+- A **service account** is an account set up within your GCP project. Authentication is handled by 
```

Comment:
This sentence is unclear, because it uses passive voice:

`A **service account** is an account set up within your GCP project.`

To make it clear that the reader doesn't have to set up the account, change the sentence to this:

`A **service account** is an account that the Kubeflow deployment process sets up within your GCP project.`

Reply:
Here I'm talking about the ways to authenticate the gcloud command to a GCP account, so the service account won't be created automatically. I'll try to make it more clear